### ISSUE TITLE
thedisconnect.co

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#914]
### Basic Information

**Test link:** `https://thedisconnect.co/three/camera-above-the-classroom/`
**Category:** other
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0
**Extension:** Nano Defender 15.0.0.118

### Additional Message

The page only displays when disconnected from internet. While a cool idea this is annoying in practice.

I think this site can probably be fixed to be readable both online and offline by removing event listener for `online` but can't get a working filter myself

**Comment by mapx-** [comment]
```
thedisconnect.co###notice-root
thedisconnect.co##*:style(filter: none !important)
```

**Comment by mapx-** [comment]
as annoyance

**Comment by yourduskquibbles** [comment]
@mapx- That only fixes the first page of content, need to disconnect from internet to get the full article to appear (multiple pages).

**Comment by mapx-** [comment]
I see .. so, the site loads all the pages when online and then you can read the page offline ?

**Comment by yourduskquibbles** [comment]
Yes it's interesting design choice by the site, but kind of annoying to go offline to read an article.

**Comment by mapx-** [comment]
I disconnected my notebook (change adapter options, ethernet, disable) => I get no cache content. Is all about only wifi connections ?

**Comment by yourduskquibbles** [comment]
Must be? Their [site .js](https://thedisconnect.co/three/static/js/main.6c23cada.js) has a lot of references to online & offline. I can't figure out on my end how to set the event handlers to always be 'offline' though. 

They also have some code in that js to test for faking offline `key:"isFakingOffline"` and checking localhost from what I can tell when looking over it. I'm not good with ##+js script inject filters - I tried for a bit to find a way to fool their test but couldn't come up with anything that worked.

**Comment by jspenguin2017** [comment]
`t: lesser`: Offline wall
